### ISSUE TITLE
unable to create production service with idling disabled

### ISSUE THREAD

**Issue by pseidel-kcf** [ClickHouse/terraform-provider-clickhouse#43]
I want to create a production service with idle_scaling disabled but I'm running into issues.  

Here is the snippet I'd like to use

```
resource "clickhouse_service" "clickhouse" {
  cloud_provider = "aws"
  region         = var.aws_region
  ip_access      = [
    {
      source      = "0.0.0.0/0"
      description = "Anywhere"
    }
  ]
  name     = "${local.full_name}-timeseries"
  tier     = "production"
  password = random_password.clickhouse.result

  min_total_memory_gb  = 24
  max_total_memory_gb  = 96
  idle_scaling         = false

  lifecycle {
    ignore_changes = [password]
  }
}
```

but this gives me an error because `idle_timeout_minutes` is required even if idle_scaling is disabled.

```
│ Error: Invalid Configuration
│ 
│   with module.main.clickhouse_service.clickhouse,
│   on ../../main.tf line 36, in resource "clickhouse_service" "clickhouse":
│   36: resource "clickhouse_service" "clickhouse" {
│ 
│ idle_scaling, min_total_memory_gb, max_total_memory_gb, and idle_timeout_minutes must be defined if the service tier is production
╵
Releasing state lock. This may take a few moments...
```

Fine I'll set idle_timeout_minutes

```
resource "clickhouse_service" "clickhouse" {
  cloud_provider = "aws"
  region         = var.aws_region
  ip_access      = [
    {
      source      = "0.0.0.0/0"
      description = "Anywhere"
    }
  ]
  name     = "${local.full_name}-timeseries"
  tier     = "production"
  password = random_password.clickhouse.result

  min_total_memory_gb  = 24
  max_total_memory_gb  = 96
  idle_scaling         = false
  idle_timeout_minutes = 0

  lifecycle {
    ignore_changes = [password]
  }

}
```

then here's the plan from the apply

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.main.clickhouse_service.clickhouse will be created
  + resource "clickhouse_service" "clickhouse" {
      + cloud_provider          = "aws"
      + endpoints               = (known after apply)
      + iam_role                = (known after apply)
      + id                      = (known after apply)
      + idle_scaling            = false
      + idle_timeout_minutes    = 0
      + ip_access               = [
          + {
              + description = "Anywhere"
              + source      = "0.0.0.0/0"
            },
        ]
      + last_updated            = (known after apply)
      + max_total_memory_gb     = 96
      + min_total_memory_gb     = 24
      + name                    = "data-access-layer-indicator-data-timeseries"
      + password                = (sensitive value)
      + private_endpoint_config = (known after apply)
      + private_endpoint_ids    = []
      + region                  = "us-west-2"
      + tier                    = "production"
    }

Plan: 1 to add, 0 to change, 0 to destroy.
```

Then I get an error from Terraform and the service has been created with idle scaling enabled and the default timeout.

```
│ Warning: Applied changes may be incomplete
│ 
│ The plan was created with the -target option in effect, so some changes requested in the configuration may have been ignored and the output values may not be fully updated. Run the following command to verify
│ that no other changes are pending:
│     terraform plan
│ 	
│ Note that the -target option is not suitable for routine use, and is provided only for exceptional situations such as recovering from errors or mistakes, or when Terraform specifically suggests to use it as
│ part of an error message.
╵
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to module.main.clickhouse_service.clickhouse, provider "module.main.provider[\"registry.terraform.io/clickhouse/clickhouse\"]" produced an unexpected new value: .idle_scaling: was
│ cty.False, but now cty.True.
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to module.main.clickhouse_service.clickhouse, provider "module.main.provider[\"registry.terraform.io/clickhouse/clickhouse\"]" produced an unexpected new value: .idle_timeout_minutes: was
│ cty.NumberIntVal(0), but now cty.NumberIntVal(15).
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
Releasing state lock. This may take a few moments...
```

Below is the output when I try to plan again

```
  # module.main.clickhouse_service.clickhouse is tainted, so must be replaced
-/+ resource "clickhouse_service" "clickhouse" {
      ~ endpoints            = [
          - {
              - host     = "axlcviofnv.us-west-2.aws.clickhouse.cloud" -> null
              - port     = 9440 -> null
              - protocol = "nativesecure" -> null
            },
          - {
              - host     = "axlcviofnv.us-west-2.aws.clickhouse.cloud" -> null
              - port     = 8443 -> null
              - protocol = "https" -> null
            },
        ] -> (known after apply)
      ~ id                   = "0e4251b7-1506-4575-b153-84a10a8ffc71" -> (known after apply)
      ~ idle_scaling         = true -> false
      ~ last_updated         = "Friday, 09-Feb-24 11:19:34 EST" -> (known after apply)
        name                 = "data-access-layer-indicator-data-timeseries"
        # (8 unchanged attributes hidden)
    }
```

I tried this first with provider v`0.0.3` but also with the latest v`0.0.5-beta.7`

**Comment by cdipesa-stord** [comment]
I am hitting this as well with latest v0.0.5-beta.7

**Comment by Kinzeng** [comment]
this should be fixed in the latest release `v0.0.5`. thanks for reporting this and please let us know if you run into any issues
